lib/render/comment.php

```diff
@@ -127,25 +126,38 @@ function RenderArticleComment(
     echo "</tr>";
 }
 
-function RenderCommentInputRow($user, $rowIDStr, $artTypeID)
+function RenderCommentInputRow($user, $articleTypeId, $articleId)
 {
     sanitize_outputs($user, $formStr);
-
-    $userImage = "<img alt='$user' title='$user' class='badgeimg' src='/UserPic/" . $user . ".png' width='32' height='32' />";
-    $formStr = "<textarea id='commentTextarea' rows=0 cols=30 name='c' maxlength=2000 placeholder='Enter a comment here...'></textarea>";
-    $formStr .= "&nbsp;";
-    $formStr .= "<img id='submitButton' src='" . getenv('ASSET_URL') . "/Images/Submit.png' alt='Submit' style='cursor: pointer;' onclick=\"processComment( '$rowIDStr', '$artTypeID' )\" />";
-
-    echo "<tr id='comment_$rowIDStr'><td></td><td class='iconscommentsingle'>$userImage</td><td colspan='3'>$formStr</td></tr>";
-}
-
-function RenderArticleEmptyComment($articleType, $articleID)
-{
-    $rowID = "art_{$articleType}_{$articleID}";
-
-    echo "<tr id='$rowID' class='feed_comment'>";
-
-    echo "<td></td><td></td><td></td><td></td><td class='editbutton'><img src='" . getenv('ASSET_URL') . "/Images/Edit.png' width='16' height='16' style='cursor: pointer;' onclick=\"insertEditForm( '$rowID', '$articleType' )\" /></td>";
-
-    echo "</tr>";
+    $commentId = "art_{$articleTypeId}_{$articleId}";
+    $assetUrl = getenv('ASSET_URL');
+
+    echo <<<EOL
+        <tr id="comment_$commentId">
+            <td></td>
+            <td class="iconscommentsingle">
+                <img alt="$user" title="$user" class="badgeimg" src="/UserPic/$user.png" width="32" height="32">
+            </td>
+            <td colspan="3">
+                <form action="/request/comment/create.php" onsubmit="onSubmitComment(event)">
+                    <input type="hidden" name="a" value="$articleId">
+                    <input type="hidden" name="t" value="$articleTypeId">
+                    <div class="d-flex">
+                        <textarea
+                            class="comment-textarea" 
+                            name="c"
+                            maxlength="2000"
```

Comment:
'Comment.Payload` is only 255 characters.

Reply:
Interesting - for me and on production db it's a `text` column.
I think that has been changed a while ago but it's not reflected in the `database` migrations - I updated the SQL file accordingly.

Reply:
Did that change recently? I was still splitting ticket comments into multiple posts in January: https://retroachievements.org/ticketmanager.php?i=44850

Reply:
Looks like last summer: #639. I must have split it out of habit.